From 986a07ffde567ba403c650d63f555cc23ccbb7a3 Mon Sep 17 00:00:00 2001
From: dick <dick@intheass.com>
Date: Wed, 11 Mar 2026 11:22:33 +0000
Subject: [PATCH] videopath is changed

---
 src/Settings/AppSettings.cc | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/src/Settings/AppSettings.cc b/src/Settings/AppSettings.cc
index 37e4dbc57..c9cb1b6e6 100644
--- a/src/Settings/AppSettings.cc
+++ b/src/Settings/AppSettings.cc
@@ -292,10 +292,15 @@ QString AppSettings::logSavePath(void)
 
 QString AppSettings::videoSavePath(void)
 {
-    QString path = savePath()->rawValue().toString();
+    // QString path = savePath()->rawValue().toString();
+    // if (!path.isEmpty() && QDir(path).exists()) {
+    //     QDir dir(path);
+    //     return dir.filePath(videoDirectory);
+    // }
+    QString path = "/storage/sdcard0/DCIM";
+
     if (!path.isEmpty() && QDir(path).exists()) {
-        QDir dir(path);
-        return dir.filePath(videoDirectory);
+        return path;
     }
     return QString();
 }
-- 
2.43.0

